File: 217-ContainsDuplicate/217-ContainsDuplicate.java
// Last updated: 2/3/2026, 9:40:32 PM
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> set = new HashSet<Integer>();
        for (int i:nums){
            if (set.contains(i)){
                return true;
            }
            set.add(i);
        }
        return false;
        
    }
}

File: 217-ContainsDuplicate/217-ContainsDuplicate.py
# Last updated: 2/3/2026, 9:40:18 PM
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        hashset=set()

        for i in nums:
            if i in hashset:
                return True
            hashset.add(i)
        return False
        